From cf5d53bf191ea8506dd6a817e46ce541a062128a Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Mon, 8 Feb 2021 06:11:07 -0800
Subject: [PATCH 1248/1921] arch/arm64: defconfig: add support for Microchip
 pin ctrl.

Marvell's CN9130-CRB board uses Microchip MCP23017 i2c pin
controller. The default config should include an entry for
supporting this GPIO controllers family PINCTRL_MCP23S08.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I633e2e38deaa0a398d3291a89f906b2441835fbc
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/45553
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/configs/marvell_v8_sdk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/marvell_v8_sdk_defconfig b/arch/arm64/configs/marvell_v8_sdk_defconfig
index edf3eb811ce2..06a502cb54a8 100644
--- a/arch/arm64/configs/marvell_v8_sdk_defconfig
+++ b/arch/arm64/configs/marvell_v8_sdk_defconfig
@@ -387,6 +387,7 @@ CONFIG_SPI_S3C64XX=y
 CONFIG_SPI_SUN6I=y
 CONFIG_SPI_SPIDEV=m
 CONFIG_SPMI=y
+CONFIG_PINCTRL_MCP23S08=y
 CONFIG_PINCTRL_SINGLE=y
 CONFIG_PINCTRL_MAX77620=y
 CONFIG_PINCTRL_IMX8MM=y
-- 
2.31.1

